catext/nether.md

```diff
@@ -6,6 +6,7 @@
 [Back](../README.md)
 
 * Must enter the nether.
-* Timing ends the frame the loading screen appears after entering the nether.
+* Timing ends (on the frame) when the loading screen appears upon entering the
+nether.
```

Comment:
This isn't even related to your change, but while at it this should also be capitalized
https://www.minecraft.net/en-us/updates/nether
```suggestion
* Timing ends (on the frame) when the loading screen appears upon entering The
Nether.
```